src/content/index.js

```diff
@@ -1,72 +1,141 @@
-
 'use strict';
 
 import { onError } from '../utils';
 import './index.css';
 
-const autoScrolling = {
-  tid: -1,
+const getScrollingElement = () => {
+  return document.scrollingElement
+    ? document.scrollingElement
+    : document.documentElement;
+};
+
+const defaultValues = {
+  intervalId: -1,
   x: 0,
   y: 0,
+  scrollingElement: null,
```

Comment:
@jpsouzasilva `scrollingElement` is already existed. Please change the value only.

Reply:
`scrollingElement: getScrollingElement()`? It is updated on the start function https://github.com/pinkienort/AutoScrolling/pull/16/files/00c4cec6a61aa8c5facf7a6e8a168c99a6ce4833#diff-421fcea811dc9e4c3d2d0486f63e42eaR33. I figured it might be more appropriate to call it there since it is dynamic.

Reply:
In diff view, `scrollingElement: getScrollingElement()` is removed, and `scrollingElement: null` is added. I want you to replace the value `getScrollingElement()` with `null`, to not change the line number.